tests/integration/Core/Content/Category/SalesChannel/NavigationRouteTest.php

```diff
@@ -239,6 +242,134 @@ public function testInclude(): void
         static::assertArrayNotHasKey('id', $response[0]);
     }
 
+    public function testLandingPageInternalLinkHasSeoUrl(): void
```

Comment:
all three generated test cases are valuable, but none of them has a SEO URL configured for the entity in question, so please add at least three more test cases where there is a SEO URL set up for those entities

also try to keep the tests lean, maybe we can use a data provider or something to make it easier to understand and read